### PR TITLE
Keras basic text classification tutorial: removed +1 from embeddings …

### PR DESCRIPTION
**Pull request by vaharoni**
…input dimensions

In the Tutorials > Beginner > ML basics with Keras > Basic text classification, there is the following code:
```python
max_features = 10000
sequence_length = 250

vectorize_layer = layers.TextVectorization(
    standardize=custom_standardization,
    max_tokens=max_features,
    output_mode='int',
    output_sequence_length=sequence_length)

# ...

model = tf.keras.Sequential([
  layers.Embedding(max_features + 1, embedding_dim),
  layers.Dropout(0.2),
  layers.GlobalAveragePooling1D(),
  layers.Dropout(0.2),
  layers.Dense(1)])
```

I believe the +1 in `max_features + 1` is unnecessary, as the vocabulary of `TextVectorization` layers already includes the padding and OOV tokens.

### CODE REVIEW

**Comment by 8bitmp3** [comment]
@MarkDaoust

**Review by MarkDaoust — APPROVED** [review]

**Comment by 8bitmp3** [comment]
@vaharoni Thanks again for the PR. Can you check the `cla/google` link below and sign the CLA is you haven't done so already  https://github.com/tensorflow/docs/actions/runs/5977063575/job/16982580897?pr=2256

**Comment by vaharoni** [comment]
I already have signed the CLA. The page now says "It looks like you've already signed this CLA. If you'd like to edit your contact information, you may do so below." I clicked on "I agree" again nevertheless. Let me know if you need me to do anything else.

**Comment by MarkDaoust** [comment]
@google-cla try again

**Comment by MarkDaoust** [comment]
Sorry about the delay, we should be good to go now.
